src/containers/shared/hooks.ts

```diff
@@ -1,3 +1,4 @@
 import { useTranslation } from 'react-i18next'
 
-export const useLanguage = () => useTranslation().i18n.resolvedLanguage
+export const useLanguage = () => useTranslation().i18n.resolvedLanguage;
```

Comment:
after @ledhed2222 commit this should not have a colon.  I don't know why it passes linting though.

Reply:
Figured out why this doesn't fail the PR.

The PR runs `lint:js` which has `--fix`.  This causes these issues to be fixed but also means there are unstaged changes so the code does not reflect this in the repo.  One solution is to remove `--fix` the other is to do a check for unstaged changed and failed the build if those exist.  Thoughts @mvadari @ledhed2222 .

Reply:
We should add a new `lint:ci` script that doesn't call `--fix`.

Reply:
i agree that the linter shouldn't fix by default. i would prefer just no `--fix` and for users to call `npm run lint -- --fix` on their own

Reply:
i didn't want to make that change in my previous PR to keep the footprint down